applications/IOIOConnectionTesterAndroid/src/main/java/ioio/connection_tester/IOIOConnectionTesterActivity.java

```diff
@@ -80,15 +80,15 @@ private void createResultListener(final View view, final TestResults results) {
 		new java.lang.Thread() {
 			@Override
 			public void run() {
-				final TextView upThroughputTextView = (TextView) view
+				final TextView upThroughputTextView = view
```

Comment:
I'm curious why ytai had all these seemingly unnecessary casts. You know of anything in past versions of Android that would have warranted all of this?

Reply:
Sorry, I can only write about the present.